src/Core/HtmlUtil.cs

```diff
@@ -374,26 +371,24 @@ public static string RemoveHtmlTags(string s, bool alsoSsaTags)
                 return null;
 
             s = RemoveHtmlTags(s);
-
-            if (alsoSsaTags)
-                s = Utilities.RemoveSsaTags(s);
-
-            return s;
+            return alsoSsaTags ? Utilities.RemoveSsaTags(s) : s;
         }
 
         public static bool IsUrl(string text)
         {
-            if (string.IsNullOrWhiteSpace(text) || text.Length < 6 || !text.Contains(".") || text.Contains(" "))
+            if (string.IsNullOrWhiteSpace(text) || text.Length < 6 || !text.Contains('.') || text.Contains(' '))
                 return false;
 
             var allLower = text.ToLower();
-            if (allLower.StartsWith("http://") || allLower.StartsWith("https://") || allLower.StartsWith("www.") || allLower.EndsWith(".org") || allLower.EndsWith(".com") || allLower.EndsWith(".net"))
-                return true;
-
-            if (allLower.Contains(".org/") || allLower.Contains(".com/") || allLower.Contains(".net/"))
+            if (allLower.StartsWith("http://", StringComparison.Ordinal) ||
```

Comment:
`text.ToLower(CultureInfo.InvariantCulture)` might be slightly more efficient too. (not tested)